styling-and-appearance/styling-apperance-editing-control-templates.md

```diff
@@ -10,31 +10,32 @@ position: 2
 
 # Editing Control Templates
 
+This article will show you two different approaches on how to extract and edit the default control templates of the UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} controls.
 
+* [Extracting Control Templates Manually from the Theme XAML File](#extracting-control-templates-manually-from-the-theme-xaml-file)
+* [Extracting Control Templates Using Visual Studio](#extracting-control-templates-using-visual-studio)
 
-Using Implicit Styles gives you the ability to easily extract and edit the default ControlTemplates of the controls. This article will show you two different approaches on how to extract the ControlTemplates.        
-      
+## Extracting Control Templates Manually from the Theme XAML File
 
-## Extracting Control Templates Manually from the Theme XAML file
+Inside the installation folder of your version of UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} you can find a folder named __Themes.Implicit__. It contains the **XAML** files of the different themes for all the controls.
 
-In the installation folder of your version of UI for {% if site.site_name == 'Silverlight' %}Silverlight{% endif %}{% if site.site_name == 'WPF' %}WPF{% endif %} you can find a folder named __Themes.Implicit__. It contains the XAML files of the different Themes for all the controls. So you basically have to navigate to the required Theme and open the needed XAML file in Visual Studio. For example if you are using the OfficeBlack Theme and you need the ControlTemplate of RadListBox you should go in the OfficeBlack folder, then into Themes folder and find the __Telerik.Windows.Controls.xaml__ file.
-![styling-apperance-editing-control-templates 1](images/styling-apperance-editing-control-templates_1.png)
+You then have to navigate to the required theme and open the needed XAML file with your favorite editor. For example, if you are using the **Office_Black** theme and you need the control template of the **RadListBox** control, you should go in the **OfficeBlack** folder and find the __Telerik.Windows.Controls.xaml__ file which usually corresponds to the name of the assembly the control is located in.

```

Comment:
The path to the folder with the themes is Themes.Implicit/OfficeBlack/Themes. You can include some informarion about the Themes folder also. 

We should also update the attached picture because its missing this information. But I think that's not that important.